### PR TITLE
Arthur branch

### PR DESCRIPTION
**Pull request by devgithubs**
# Business Analysis 🟠 Code Institute 🟠 Community Project - Pull Request 🎉🎉

---

👉 _**Please remove this banner, and the below content under each header and replace with your own values, leaving the headers where they are.**_ 👈

---

## Pull Request References:
- PLEASE IDENTIFY THE ISSUE/FEATURE/TASK THIS PR IS ASSOCIATED WITH. E.G:
    - #U01 👈 Notice the `#`, followed by the Prefix Category of `U`, and then the feature number `01`
    - If you are not sure where to find the issue, feel free to use the Direct URL to the Issue itself.

## PR contains:
- INFO ABOUT YOUR PULL REQUEST GOES HERE _(Please be as descriptive as possible)_ 🤜
- DETAIL INFORMATION ABOUT THE WORKS DONE TO SATISFY THE TASK/FEATURE/ISSUE.
- FEEL FREE TO USE CHECKBOX ITEMS TO DENOTE THE WORK CARRIED OUT IN THIS PR.
    - An easy to follow markdown cheatsheet can be found [here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet). 📃📃

## Breaking changes
- IF ANYTHING YOU'RE COMMITTING WOULD BREAK SOMETHING, INCLUDE HERE WHAT WOULD BREAK
- IF ANYTHING YOU'RE COMMITTING WOULD ALTER SOMETHING DRASTICALLY THAT WAS PREVIOULSY WORKING AS EXPECTED, INCLUDE THAT HERE.
- IF YOU HAVE NO BREAKING CHANGES, ENTER 'None'

## Screenshots/Gifs/Media
- IF YOU HAVE ANY SCREENSHOTS, INCLUDE THEM HERE.
- IF YOU HAVE ANY MEDIA (GIFS/MP4) TO SHOWCASE A FEATURE, INCLUDE THEM HERE.
    - Media can be added in the form Gifs/MP4s/PNGs/JPEGs etc. Drag and drop them in to see them appear. Use the "Preview" tab to view your media and make sure it has come over as expected.
- IF YOU HAVE NO SCREENSHOTS, ENTER 'None'

### CODE REVIEW

**Review by auxfuse — CHANGES_REQUESTED** [review]
Hi @devgithubs just wondering, about some things to do with the webScraper. Are we going to eventually have a form that feeds into the function from a particular type of user to then check and "scrape" that page?
@stefdworschak giving you a tag in this, any thoughts or considerations to be made from this before we Approve/Merge?

**Review by stefdworschak — CHANGES_REQUESTED** [review]
Please have a look at my comment and adjust the file before we merge. Thanks

**Comment by stefdworschak** [comment]
@devgithubs great to see you made the requested changes to the PR. Just one thing, it seems that you have now completely removed the `getHTMLdocument` function? It is still used in `app/categories.py`, so best if you just add that function back in. Maybejust above your newly created function.

File: app/functions/company_web_scraper.py [hunk 1]
```diff
@@ -2,41 +2,40 @@
 from bs4 import BeautifulSoup
 import re
 
-
+def scraper(url_to_scrape):
```
[the remark this answers is not in the document]
Comment (stefdworschak) [reply app/functions/company_web_scraper.py:12]: Updated as requested.

**Review by stefdworschak — APPROVED** [review]
LGTM

**Pull request merged**